infrastructure/containerized-data-importer/README.md

```diff
@@ -0,0 +1,13 @@
+## Containerized data importer
+Is a persistent storage management add-on for Kubernetes. It's primary goal is to provide a declarative way to build Virtual Machine Disks on PVCs for Kubevirt VMs
+
+## How to install
+In the following you can find the commands you need to execute in order to deploy CDI in a Kubernetes cluster:
+```bash
+#Export last cdi version
+export VERSION=$(curl -s https://github.com/kubevirt/containerized-data-importer/releases/latest | grep -o "v[0-9].[0-9].[0-9]")
+#Deploy the cdi operator
```

Comment:
What's the difference between 'cdi-operator' and 'cdi-cr'?

Reply:
Like in kubevirt before we deploy the operator and after the cr (instance deployment request) which triggers the actual installation

Reply:
we will change the description, maybe with "#Deploy cdi CR which triggers the actual installation"

Reply:
The last one looks good to me.